packages/modular-scripts/src/init.ts

```diff
@@ -0,0 +1,49 @@
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import execSync from './utils/execSync';
+import { ModularPackageJson } from './utils/isModularType';
+
+export async function initModularFolder(
+  folder: string,
+  initOverride: boolean,
+): Promise<void> {
+  const packageJsonPath = path.join(folder, 'package.json');
+
+  if (!fs.existsSync(packageJsonPath)) {
```

Comment:
If this project hasn't been initiated yet, why not port over the package.json template from CMRA?

Reply:
Yeah - good point we could actually just make CMRA call modular-scripts init?